manifests/params.pp

```diff
@@ -15,7 +15,7 @@
   $sources                = undef
   $puppet_master          = true
   $proxy                  = undef
-  $pool_size              = undef
+  $pool_size              = $facts['processors']['count']
```

Comment:
often one does `cpu.count - 1` to not completely utilize the system. 
is this a thing we should consider here to?

Reply:
I don't think we need to consider it. My impression is that this is mostly IO bound or limited by the performance forge.puppet.com delivers.

Reply:
okay. 👍  just came to my mind when i saw this change 🤔